api-lib/gql/__generated__/zeus/const.ts

```diff
@@ -2204,15 +2204,18 @@ export const AllTypesProps: Record<string, any> = {
     _not: 'colinks_gives_bool_exp',
     _or: 'colinks_gives_bool_exp',
     activity_id: 'bigint_comparison_exp',
+    attestation_uid: 'String_comparison_exp',
     cast_hash: 'String_comparison_exp',
     created_at: 'timestamptz_comparison_exp',
     give_skill: 'skills_bool_exp',
     giver_profile_public: 'profiles_public_bool_exp',
     id: 'Int_comparison_exp',
+    onchain_sync_error: 'String_comparison_exp',
```

Comment:
maybe we want on onchain_synced_at timestamp

Reply:
ok, will add and use as primary search for things to sync